core/src/event_id.rs

```diff
@@ -1,100 +1,70 @@
+//! EventId generates EventIDs from event data.
+//!
+//! varint(0xce) + // streamid, 1 byte
+//! varint(0x05) + // cip-124 EventID, 1 byte
+//! varint(networkId), // 1 byte (5 for local network)
+//! last8Bytes(sha256(separator_value)), // 16 bytes
+//! last8Bytes(sha256(stream_controller_DID)), // 16 bytes
+//! last4Bytes(init_event_CID) // 8 bytes
+//! cbor(eventHeight), // 1-3 bytes
+//! eventCID // 36 bytes
+//!   0x01 cidv1, 1 byte
+//!   0x71 dag-cbor, 1 byte
+//!   0x12 sha2-256, 1byte
+//!   0x20 varint(hash length), 1 byte
+//!   hash bytes, 32 bytes
 #![warn(missing_docs, missing_debug_implementations, clippy::all)]
 
 use cbor::Encoder;
-use cid::Cid;
-use multihash::{Hasher, Sha2_256};
+use cid::{
+    multihash::{Hasher, Sha2_256},
```

Comment:
Why is it better to import multihash from cid then directly?

Reply:
This guarantees we are using a compatible version of multihash with the cid crate.